### PR TITLE
Create PluginsPane component

### PR DESCRIPTION
**Pull request by ismail-syed**
* Created a `PluginsPane` component that can be used to create multiple panes which can be controlled through tabs

screen shot 2018-01-15 at 1 00 48 am

cc/ @justinkrol @DerekStride

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -52,7 +52,8 @@
   "dependencies": {
     "codemirror": "^5.26.0",
     "codemirror-graphql": "^0.6.11",
-    "marked": "0.3.6"
+    "marked": "0.3.6",
+    "react-tabs": "^2.2.1"
```
Comment (ismail-syed) [inline package.json:56]: I'm using the [`react-tabs`](https://github.com/reactjs/react-tabs) to handle all tabbable UI stuff.

File: src/components/GraphiQL.js [hunk 2]
```diff
@@ -16,10 +16,10 @@ import { ToolbarButton } from './ToolbarButton';
 import { ToolbarGroup } from './ToolbarGroup';
 import { ToolbarMenu, ToolbarMenuItem } from './ToolbarMenu';
 import { ToolbarSelect, ToolbarSelectOption } from './ToolbarSelect';
+import { PluginsPane } from './PluginsPane';
 import { QueryEditor } from './QueryEditor';
 import { VariableEditor } from './VariableEditor';
 import { ResultViewer } from './ResultViewer';
-import { AnalysisViewer } from './AnalysisViewer';
```
Comment (ismail-syed) [inline src/components/GraphiQL.js:22]: I've removed `AnalysisViewer` for now

**Comment by justinkrol** [comment]
This is great! I was fiddling around with something to try and get this working this weekend. I'll see if I can incorporate resizing into this.

File: src/components/GraphiQL.js (hunk 2 above)
Comment (justinkrol) [reply src/components/GraphiQL.js:22]: Are you envisioning using the component code for this at some point? We should probably just remove it for now as well and re-add if necessary.

File: package.json (hunk 1 above)
Comment (DerekStride) [reply package.json:56]: Would the graphiql team support adding this dependency?

**Pull request merged**